pb-test/gen/pb-jelly/proto_pbtest/src/pbtest3.rs.expected

```diff
@@ -4120,10 +4119,6 @@ impl ::pb_jelly::Message for TestMessage3ErrIfDefaultEnumOneof {
         }
       }
     }
-    match something {
-      Some(v) => self.something = v,
-      None => return Err(::std::io::Error::new(::std::io::ErrorKind::InvalidInput, "err_if_default_or_unknown 'something' had no value while parsing message pbtest.TestMessage3ErrIfDefaultEnumOneof")),
-    }
     Ok(())
```

Comment:
This illustrates the fix. It is very finicky, since observing the bug relies on having random leaked values from the field ordering.